examples/mspm0g3519/MSPM0GX51X_Series.yaml

```diff
@@ -0,0 +1,424 @@
+name: MSPM0GX51X Series
+manufacturer:
+  id: 0x17
+  cc: 0x0
+generated_from_pack: true
+pack_file_release: 1.0.0
+variants:
+- name: MSPM0G1518
```

Comment:
This file should probably not be committed

Reply:
The target files are temporary until probe-rs releases a new version after which I'd remove those files.

I can remove these of course, but then you'd have to build probe-rs from git or convince probe-rs to make a new release.